en/04-preparing-your-project-being-open-sourced.md

```diff
@@ -0,0 +1,75 @@
+
+We will talk more about this important and complicated subject in a further article. If you want to get informed about its publication don't hesitate to follow our twitter [@ZenikaOSS](https://twitter.com/ZenikaOSS)!
+
+## Contributing guide
+
+> A CONTRIBUTING.md file, in your open source repository or site, provides potential project contributors with a short guide to how they can help with your project or study group. It is convention to capitalize the word "contributing" as the file title, and to save it as a resource in markdown (hence the extension .md).
+> -- [<cite>Mozilla Science Lab</cite>](https://mozillascience.github.io/working-open-workshop/contributing/)
+
+Earlier we talked about **The open source contributor funnel** in our readme we provided documentation to help *users* installing and using your project, you will now focus on *contributors*. The contributing guide is made to give instructions to everybody that want to participate on your project.
+
+Most of the time contributions are made by users because they encounter a problem using your project. It's nice to a have information on how to report a bug or suggest a new feature.
+
+If you want code contributors you need to give all details about how to set up project's development environment and how to submit a contribution. [Mocha's contributing guide](https://github.com/mochajs/mocha/blob/master/.github/CONTRIBUTING.md#shoe-contributing-code-step-by-step) is a good example of step by step instructions to get your contribution merged. 
+
+The contributing guide is also the good place to describe coding styles. You can enforce good practices such as testing or conventions like in [Immutadot's contributing guide](https://github.com/zenika-open-source/immutadot/blob/master/.github/CONTRIBUTING.md#tests-and-code-style-policeman).
+
+If your project is on GitHub this file will be [automatically linked](https://help.github.com/en/articles/setting-guidelines-for-repository-contributors) when a contributor open an issue or create a pull request.
+
+## Code of conduct
+
+> Having a CODE_OF_CONDUCT.md in your public repository lets potential *contributors* know in advance how they can expect to be treated by the community and *maintainers*.
+> -- [<cite>Michael Jolley</cite>](https://dev.to/michaeljolley/using-a-contributing-codeofconduct-to-assist-others-in-contributing-to-public-repositories-1l90)
+
+The code of conduct of your project is a document that protects every participants. It helps creating a welcoming community. 
+
+A good addition to your code of conduct is an explanation about how you plan to enforce your code of conduct. It's important to show that you take it seriously so everybody knows what action will be taken in case of a code of conduct violation.
```

Comment:
```suggestion
A good addition to your code of conduct is an explanation about how you plan to enforce it. It's important to show that you take it seriously so everybody knows what action will be taken in case of a code of conduct violation.
```